src/index.ts

```diff
@@ -92,7 +92,7 @@ function newOperationFactory(config: Config, schema: GraphQLSchema, def: Operati
     export function new${name}Response(
       ${hasVariables ? code`variables: ${maybeImport(config, `${name}${operation}Variables`)},` : ""}
       data: ${name}DataOptions | Error
-    ): MockedResponse<${name}${operation}Variables, ${name}${operation}> {
+    ): MockedResponse<${maybeImport(config, `${name}${operation}Variables`)}, ${name}${operation}> {
```

Comment:
This was really the only change necessary? :thinking: If `newFooResponse` ends up in `graphqlFactories.tsx`, how does `maybeImport` know which of the many near-filed `*.generated.ts` files to import for? I didn't think it was that smart.

...or is near-file support a separate PR, and this is just fixing separate factory file? That'd makes sense...

Reply:
Ah, yeah this PR doesn't tackle that. See https://github.com/homebound-team/graphql-typescript-response-factories/pull/35 for the `near-operation-file` change.

Once I merge this, I will rebase the other one. It got all confused with the stacking.